### DISCUSSION TITLE
Items accessed from insights dashboard have missing data

### DISCUSSION THREAD

**Discussion by ldrach** [directus/directus#13785] [Archive]
I'm not sure if this is a bug or if I messed some setting up somewhere along the way.

For context, I have a registration form that has M2O relationships to other collections for the content of each section (eg. Student info, Address, etc.)

If I go to a submitted form from the Content tab and look into the different items, the data is displayed as expected. In this case below, there is data for first name, last name, DOB, primary phone, and gender.

![image](https://user-images.githubusercontent.com/40499729/172660067-63bcaf51-cbe0-4549-baaf-eb39625b652a.png)

I wanted to set up a dashboard in insights that would give a clerical user a more specialised interface. Setting everything up went really well; however, when I was looking into the same student item but from this new dashboard in insights, not all of the data is being displayed.

![image](https://user-images.githubusercontent.com/40499729/172661235-f6876a30-51c1-4523-910a-eacfe770ddcd.png)

![image](https://user-images.githubusercontent.com/40499729/172661339-659a3d19-6bbc-491d-939e-faf86c6e6d35.png)

Now the same student item only displays first name and last name.

It seems that as far as Directus is concerned, those fields are actually empty even though I can see the data in the database. Some of the fields are required fields like first name, last name, DOB, primary phone, and gender. If I use this view to try to input just a middle name and save, I get an error because there is no data in the required fields.

![image](https://user-images.githubusercontent.com/40499729/172662174-cd9232b3-3868-458b-9594-92617d9d6668.png)

If I were to do the same thing in the content view that displays the data correctly, I can save normally since the data in those fields is being displayed. 

I thought this could be a permission issue at first but I don't think that would be the cause since I am having this issue as an Admin user. Thought I would see if anyone else has had an issue like this or had any ideas for possible solutions since I cannot find any reason why this would be happening.

Thanks!

**Comment by azrikahar** [comment]
What version of Directus are you currently on? When you open the M2O student record, it shows "Creating Item in Student" and that does indicate something is not right. However, this does seem to be similar to #13106, which should have been fixed by #13151 on Directus `9.11.0` onwards. 

In case you are already on the latest version at the time of writing (which is `9.12.1`), it'd be very helpful if you can replicate this issue on a minimal sqlite file and share it here with the credentials 👍

**Reply by ldrach** [reply]
Currently I am using Directus 9.12.2. I can look at replicating it with sqlite when I have some time to do so!

More recently I tried creating a panel directly for the 'Student' collection which seems to work fine. It only seems to not work right when I make the panel for 'Registration Form' which has the M2O relationship to the other collections like Student. I checked all the other collections I have M2O relationships to in the Registration Form and they all seem to function properly and have "Editing Item in <collection>" at the top as expected. I even remade the Student collection and then made a new M2O relationship in Registration Form but still no luck.

When I have some more time to troubleshoot this I can post some updates
